tests/nsrt_learning/strips_learning/test_gen_to_spec_learner.py

```diff
@@ -27,7 +31,7 @@ def find_unification(necessary_add_effects,
 
 
 def test_backchaining_strips_learner():
-    """Test the BackchainingSidePredicateLearner."""
+    """Test the BackchainingSTRIPSLearner."""
```

Comment:
good catch. the word "BackchainingSidePredicateLearner" appears in one other place in this file, can you fix that too?